modules/EventEmitter.js

```diff
@@ -654,7 +654,7 @@ class EventEmitter {
         } = this.ctx;
 
         this._on('kad-data-location-request', async (kadMessage) => {
-            logger.info('kad-data-location-request received');
+            logger.debug('kad-data-location-request received');
```

Comment:
Can we add more details to all these messages? Who sent it? For which import maybe? I am not sure what is available in kadMessage, but we should have more data.